nuxt-app/pages/profile-creation.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import Pagination from '~/components/Pagination.vue'
+import ProfileCreationDetails from '~/components/ProfileCreationDetails.vue'
+import ProfileCreationEmojis from '~/components/ProfileCreationEmojis.vue'
+import ProfileCreationInterests from '~/components/ProfileCreationInterests.vue'
+import ProfileCreationMainInfos from '~/components/ProfileCreationMainInfos.vue'
+import { useDirectus } from '~/composables/useDirectus'
+import type { DirectusProfileCreationPage } from '~/types'
+import { computed, ref, type ComputedRef } from 'vue'
+import { useLoadingScreen } from '../composables'
+import { getMetaInfo } from '../helpers'
+
+const directus = useDirectus()
+
+const { data: pageData } = useAsyncData(async () => {
+    const [profileCreationPage, tags] = await Promise.all([directus.getProfileCreationPage(), directus.getAllTopTags()])
+    return { profileCreationPage, tags }
+})
+
+const profileCreationPage: ComputedRef<DirectusProfileCreationPage | undefined> = computed(
+    () => pageData.value?.profileCreationPage
+)
+const tags = computed(() => pageData.value?.tags)
+
+useLoadingScreen(profileCreationPage)
+
+useHead(
+    getMetaInfo({
+        type: 'website',
+        path: '/profile-creation',
+        title: 'Profile Creation',
```

Comment:
Sollten wir sowas eher auf Deutsch machen? (Weil auch in Suchmaschinen angezeigt etc.?)